sdk/ai/azure-ai-assistants/tests/test_model_assistants_client.py

```diff
@@ -0,0 +1,706 @@
+# originally copied from azure-ai-inference 
+
+# # ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+import os
+import json
+import azure.ai.assistants as sdk
+
+from model_assistants_test_base import (

```

Comment:
Maybe model is inherited from inference sdk tests. Assistants is a bit different construct, we can remove model reference